multichain-testing/test/account-balance-queries.test.ts

```diff
@@ -44,7 +44,7 @@ const queryAccountBalances = test.macro({
       retryUntilCondition,
     } = t.context;
 
-    const agoricAddr = wallets[chainName];
+    const agoricAddr = wallets[walletName];
     const wdUser1 = await provisionSmartWallet(agoricAddr, {
```

Comment:
We might consider moving `provisionSmartWallet` to `test.before()` and reusing it across runs. For now, this is expedient.